core/cloudflow-operator/src/main/scala/cloudflow/operator/runner/SparkRunner.scala

```diff
@@ -129,7 +129,8 @@ object SparkRunner extends Runner[CR] with PatchProvider[SpecPatch] {
     val name = resourceName(deployment)
     val labels = appLabels.withComponent(name, CloudflowLabels.StreamletComponent) + ("version" -> "2.4.5") ++
           updateLabels ++
-          Map(Operator.StreamletNameLabel -> deployment.streamletName, Operator.AppIdLabel -> appId).mapValues(Name.ofLabelValue)
+          Map(Operator.StreamletNameLabel -> deployment.streamletName, Operator.AppIdLabel -> appId).mapValues(Name.ofLabelValue) ++
+          getLabels(podsConfig, PodsConfig.CloudflowPodName)
```

Comment:
Same issue here if the specify in driver and/or executor named pod. https://cloudflow.io/docs/current/develop/cloudflow-configuration.html#_kubernetes_container_settings

Reply:
Done